test/ViewQuery/Dimensions.test.js

```diff
@@ -0,0 +1,81 @@
+const { strictEqual } = require('assert')
+const { describe, it } = require('mocha')
+const rdf = require('rdf-ext')
+const clownface = require('clownface')
+
+const ns = require('../support/namespaces')
+const Dimensions = require('../../lib/query/ViewQuery/Dimensions.js')
+const { Parser } = require('n3')
+
+describe('.resolveProperty', () => {
```

Comment:
Then, I would not longer test an "internal" function but rather focus on the public API of the `Dimensions` class

Reply:
You're right! makes sense